docs/creating-new/create/cmake.rst

```diff
@@ -456,6 +456,23 @@ To locally check if the documentation is still building you can run:
   [hunter/docs]> source ./jenkins.sh
   (_venv) [hunter/docs]> ./make.sh
 
+If the documentation contains spelling errors or unrecognized names, the documentation test build will fail and report the unrecognized strings.  Fix any spelling errors and test the build again.  Any remaining errors can be fixed by adding all correct but unrecognized names, string, or terms to the ``spelling`` header at the top of the document entry ``docs/packages/pkg/unrecognized-name.rst``.  In this example, ``unrecognized-name`` would be a package name that is not in the dictionary.
+
+.. code-block:: none
+  :emphasize-lines: 1-4
+  
+  .. spelling::
+
+  unrecognized-name
```

Comment:
I think they should be added separately, follow-up: https://github.com/ruslo/hunter/commit/65678c0b8fd8526c9a4256d18e3ce1310e3daa54